requirements-dev.txt

```diff
@@ -5,4 +5,5 @@ pyyaml
 responses
 backports.tempfile
 mock
+unittest2
```

Comment:
Why is this necessary? Is there something missing from the Python 2.7 stdlib unittest?

Reply:
Python3 added TestCase.subTest() which is very useful and this backport has it too.